src/containers/Projects/components/SideMenu/index.tsx

```diff
@@ -26,10 +26,7 @@ const SideMenu: FC<Props> = ({breadcrumbHeight}) => {
     <div className="ProjectsSideMenu">
       <div className="ProjectsSideMenu__section">
         <button
-          className={clsx(
-            'ProjectsSideMenu__section-header',
-            isProjectRulesSelected && 'ProjectsSideMenu__section-header--active',
```

Comment:
please include this back into the className here

Reply:
okay